pgcli/main.py

```diff
@@ -553,6 +580,19 @@ def need_completion_refresh(queries):
 
     return False
 
+
+def has_change_db_command(queries):
```

Comment:
Can you change https://github.com/dbcli/pgcli/pull/379/files#diff-38f3aeda57742756afa677ca09dcea43R575 to use this new function? So we don't duplicate the same logic twice?